stdpopsim/species.py

```diff
@@ -169,6 +170,8 @@ def get_contig(
             same chromosome-specific mutation and recombination rates.
             This option cannot currently be used in conjunction with the
             ``genetic_map`` argument.
+        :param float mutation_rate: The per-base mutation rate. If none is given,
```

Comment:
Why do we need a species mutation rate as well?

Reply:
Previously, the QC model (just the Demography object) was attached to a model to check 1) that a model has been QC'd (is not None), and 2) to check for equivalency in tests. I think it would also be good to check equivalency of other attributes of the model to its QC counterpart, such as generation time and mutation rate, so that those are more easily available to compare to each other if the full QC stdpopsim DemographicModel object is attached instead.

Reply:
I think this comment is about the ``qc_model`` -> ``qc`` change above? (I don't see the connection with Species.mutation_rate?)

Reply:
Yeah sorry, replied to wrong review.. edited comment.

Reply:
Hmm - Github seems to have messed up the comments here - ignore

Reply:
Hmm, well, let's take the Species.mutation_rate out of this PR and get the other changes merged first. It's not clear why we'd have a Species.mutation_rate and not a Species.recombination_rate, etc, and why this wouldn't be derived from the per-chromosome values that we're trying to catalog.

Reply:
This is in the `get_contig` function, so that you can specify a contig with the demographic model's mutation rate instead of the chromosome rate. I don't think there is a Species.mutation_rate still?

Reply:
Ah, sorry, my bad.